index.html

```diff
@@ -59,7 +59,7 @@ <h4 class="jumbotron-text">Project Jupyter exists to develop open-source softwar
                         <img class="img-responsive" src="assets/labpreview.png" id="portal" alt="examples of jupyterlab workspaces in single document and multiple document workspaces"/>
                     </div>
                     <div class="col-md-6 nb-highlight-text">
-                        <h3>JupyterLab 1.0: Jupyter’s Next-Generation Notebook Interface</h3>
+                        <h3>JupyterLab 2.0: Jupyter’s Next-Generation Notebook Interface</h3>
```

Comment:
Thanks! Can we just remove the version number now? The 1.0 version was a big enough milestone we wanted to emphasize it, but now the version numbers will get out of date much more frequently. For example, 3.0 is coming out very soon.

Reply:
True, we could fetch this actually so that it is always up to date.

Reply:
I think it reads better as `JupyterLab: Jupyter's Next-Generation Notebook Interface`. Otherwise, it constantly sounds like it just became the next-generation interface at the last version (e.g., JupyterLab 3.0 is the next-generation interface, but not jlab 2.0)

Reply:
Fair enough, will update!

Reply:
in other words, I think it would read better if this page now frames the jlab project, not a specific milestone of jlab

Reply:
(and it's a lot less work too :)

Reply:
Agreed :-)